src/pages/content/word-list.mdx

```diff
@@ -1,12 +1,707 @@
 ---
 title: Word list
-description: >
-  Lorem ipsum dolor sit amet, consectetur adipiscing
-  elit, sed do eiusmod tempor incididunt ut labore et
-  dolore magna aliqua.
 ---
 
-Coming soon
+## Words we use
+
+### @mention
+
+Pronounced “at mention.” Always lowercase, even at the beginning of a sentence.
+
+### accessible, accessibility, a11y
+
+Describes a set of standards that make our product usable to people experiencing
+disabilities. Avoid using accessible as a synonym for “simple.” For that
+purpose, use terms like easy to learn, easy to use, or intuitive.
+
+### add
+
+Use add when moving a “thing” in the product from one place to another.
+
+- Compare with [assign](/content/word-list#assign), [create](/content/word-list#create)
+
+<BestPractice>
+  <Do>
+    <ul>
+      <li>Add a user to a group.</li>
+      <li>Assign user to a group.</li>
+    </ul>
+  </Do>
+  <Dont>
+    <ul>
+      <li>Assign a ticket to this agent.</li>
+      <li>Add this ticket to the agent.</li>
+    </ul>
+  </Dont>
+</BestPractice>
+
+### assign
+
+Use assign when referring to ownership and responsibility.
+
+- Compare with [add](/content/word-list#add), [create](/content/word-list#create)
+
+### create
+
+Use create when you’re about to make a brand new thing in the product.
+
+- Compare with [add](/content/word-list#add)
+
+<BestPractice>
+  <Do>
+    <p>Create object record.</p>
+  </Do>
+  <Dont>
+    <p>Add a new object record.</p>
+  </Dont>
+</BestPractice>
+
+<!-- markdownlint-disable -->
+
+### aka, a.k.a.
+
+Stands for also known as. Use aka instead of a.k.a..
```

Comment:
This is kind of awkward, so I'll rephrase. Please include aka in italics.

Stands for also known as. Instead of a.k.a., use _aka_ with no punctuation.